libs/execution_context/ebpf_core.c

```diff
@@ -1346,7 +1345,6 @@ _ebpf_core_protocol_get_pinned_object(
     retval = ebpf_core_get_pinned_object(&path, &reply->handle);
 
 Done:
-    EBPF_OBJECT_RELEASE_REFERENCE((ebpf_core_object_t*)object);
```

Comment:
Doesn't ebpf_pinning_table_find take a reference on the object before returning it?